packages/@interaxyz/mobile/src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -35,14 +52,22 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: 'center',
     justifyContent: 'center',
+    backgroundColor: colors.backgroundPrimary,
   },
   backgroundImage: {
     ...StyleSheet.absoluteFillObject,
     resizeMode: 'stretch',
     width: undefined,
     height: undefined,
   },
-  text: {
+  textWithImage: {
+    ...typeScale.titleLarge,
```

Comment:
This font isn't rendering properly in the example app screenshots, but it is in the beefy app. So trust me :)